module/ldbc-connector/js/src/main/scala/ldbc/connector/authenticator/MysqlNativePasswordPlugin.scala

```diff
@@ -6,27 +6,10 @@
 
 package ldbc.connector.authenticator
 
-import scala.scalajs.js
-import scala.scalajs.js.typedarray.Uint8Array
+import cats.effect.Concurrent
+import fs2.hashing.Hashing
 
-import scodec.bits.ByteVector
-
-class MysqlNativePasswordPlugin extends AuthenticationPlugin:
-
-  private val crypto = js.Dynamic.global.require("crypto")
+class MysqlNativePasswordPlugin[F[_]: Hashing: Concurrent] extends AuthenticationPlugin[F]:
 
   override def name: String = "mysql_native_password"
 
-  override def hashPassword(password: String, scramble: Array[Byte]): Array[Byte] =
```

Comment:
I guess this change introduces breaking (behavior) change because new implementation uses sha256 hash for password hashing while previous implementation uses sha1 hash 🤔

Reply:
Oh...most likely it was an implementation error.
The following is an unintended behavior when working with JS/Native. If this is a mistake, testing will reveal something.
https://github.com/takapi327/ldbc/blob/master/module/ldbc-connector/shared/src/main/scala/ldbc/connector/net/Protocol.scala#L406